CMakeLists.txt

```diff
@@ -51,51 +71,67 @@ endif()
 #message(${N2N_OPTION_AES})
 ## DEBUG FOR CMAKE
 
+INCLUDE_DIRECTORIES(.)
+INCLUDE_DIRECTORIES(include)
+if(DEFINED WIN32)
+  INCLUDE_DIRECTORIES(win32)
+  INCLUDE_DIRECTORIES("D:/Program Files/MinGW/opt/include/" "D:/Program Files/MinGW/x86_64-w64-mingw32/include/")
+  LINK_DIRECTORIES("D:/Program Files/MinGW/opt/lib/" "D:/Program Files/MinGW/x86_64-w64-mingw32/lib/")
```

Comment:
I don't think it's a good idea to hardcode paths

Reply:
For Windows, the path of the header file and the library are uncertain. These two absolute paths need to be changed according to the actual situation at the time of construction. For the time being, there is no other better solution. If they are not coordinated here, can directly comment them.